poms/bom/pom.xml

```diff
@@ -0,0 +1,149 @@
+
+    <properties>
+        <cassandra-quarkus.version>1.0.0-SNAPSHOT</cassandra-quarkus.version>
+        <datastax-java-driver.version>4.5.0</datastax-java-driver.version>
+        <lz4.version>1.6.0</lz4.version>
+        <snappy.version>1.1.7.3</snappy.version>
+        <test-containers.version>1.12.4</test-containers.version>
+        <awaitility.version>4.0.2</awaitility.version>
+        <assertj.version>3.15.0</assertj.version>
+        <mockito.version>3.3.0</mockito.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- External BOM -->
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom</artifactId>
+                <version>${quarkus.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <!-- Cassandra Quarkus -->
+            <dependency>
+                <groupId>com.datastax.oss.quarkus</groupId>
+                <artifactId>cassandra-quarkus-client</artifactId>
+                <version>${cassandra-quarkus.version}</version>
+            </dependency>
+
+            <!-- Other third party dependencies here -->
+            <dependency>
+                <groupId>com.datastax.oss</groupId>
+                <artifactId>java-driver-core</artifactId>
+                <version>${datastax-java-driver.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>jcl-over-slf4j</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.codehaus.groovy</groupId>
+                        <artifactId>groovy</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.codehaus.groovy</groupId>
+                        <artifactId>groovy-json</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.squareup</groupId>
+                        <artifactId>javapoet</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.ow2.asm</groupId>
+                        <artifactId>asm-util</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>com.datastax.oss</groupId>
+                <artifactId>java-driver-mapper-runtime</artifactId>
+                <version>${datastax-java-driver.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.datastax.oss</groupId>
+                <artifactId>java-driver-mapper-processor</artifactId>
+                <version>${datastax-java-driver.version}</version>
+            </dependency>
+            <dependency>
```

Comment:
Compression libraries should be optional, no?

Reply:
yes, I've created a separate task for it:
https://datastax-oss.atlassian.net/browse/JAVA-2707